### ISSUE TITLE
onSwitch is undefined - "this" is "window", not "pjax"

### ISSUE THREAD

**Issue by darylteo** [MoOx/pjax#51]
```
Pjax switch fail:  
TypeError: this.onSwitch is not a function(…)(anonymous function) @ vendor.bundle.js:277 
````

Stack Trace 
````
module.exports.outerHTML (vendor.bundle.j…v=20160105:740)
(anonymous function) (vendor.bundle.j…v=20160105:722)
module.exports (vendor.bundle.j…v=20160105:390)
(anonymous function) (vendor.bundle.j…v=20160105:713)
module.exports (vendor.bundle.j…v=20160105:697)
Pjax.switchSelectors (vendor.bundle.j…v=20160105:174)
Pjax.loadContent (vendor.bundle.j…v=20160105:225)
(anonymous function) (vendor.bundle.j…v=20160105:272)
request.onreadystatechange (vendor.bundle.j…v=20160105:860)
````

Happens in switches.js outerHTML and innerHTML

**Comment by darylteo** [comment]
Now it's 

```
"TypeError: Pjax.forEachEls is not a function
    at vendor.bundle.js?v=20160105:239:13
    at Array.forEach (native)
    at Object.Pjax.loadContent (vendor.bundle.js?v=20160105:238:29)
    at Object.<anonymous> (vendor.bundle.js?v=20160105:272:15)
    at XMLHttpRequest.request.onreadystatechange (vendor.bundle.js?v=20160105:872:10)"
```

**Comment by darylteo** [comment]
And now it's 

```
"TypeError: Pjax.executeScripts is not a function
    at vendor.bundle.js?v=20160105:242:15
    at NodeList.forEach (native)
    at module.exports (vendor.bundle.js?v=20160105:392:37)
    at vendor.bundle.js?v=20160105:241:8
    at Array.forEach (native)
    at Object.Pjax.loadContent (vendor.bundle.js?v=20160105:240:29)
    at Object.<anonymous> (vendor.bundle.js?v=20160105:274:15)
    at XMLHttpRequest.request.onreadystatechange (vendor.bundle.js?v=20160105:874:10)"
````

Sorry for the spam, just recording these as I go along.

**Comment by darylteo** [comment]
Also clone.

**Comment by MoOx** [comment]
Closed by c12e4cdeddf48b9aec451aa745bed8d5fbb7c129 ?

**Comment by darylteo** [comment]
Yes that commit should fix the errors.

I have a parched 0.2.1 working on my app right now (bugs notwithstanding)

Daryl
